src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -87,6 +86,11 @@ function AddressPage({privatePersonalDetails, route}) {
      * @param {Object} values - form input values
      * @returns {Object} - An object containing the errors for each inputID
      */
+    useEffect(() => {
```

Comment:
There are some linter errors still
<img width="568" alt="image" src="https://github.com/Expensify/App/assets/9059945/7a7a148f-a34f-4e6b-a0b8-4f3a4e4e0d7f">

You can check those by running `npm run lint` (ignore the errors in /vendor/bundle/ruby)